code/payments/Tally-Vs-Rollup.md

```diff
@@ -0,0 +1,69 @@
+# Tally vs Rollup
+
+The terms "tally" and "rollup" are used throughout the code base. Their specific meanings have been a source of confusion since their meanings are similar.
+
+The purpose of this document is to:
+- define the terms "tally" and "rollup".
+- provide details about how tally and rollup are used in the code base.
+
+## Definitions
+
+Both tally and rollup sum an amount of something (i.e. storage or bandwidth) as it relates to a period of time.
+
+They differ in what the period of time is.
+- For tally, the period of time is just a single moment in time, typically the most recent or current moment.
```

Comment:
It's also that `tally` might be slightly wrong at a specific point in time. For example, when walking over the pointers something gets inserted before our cursor, although the "time" is after the insert, it won't be included in the tally until after the next one.